Libplanet.Action/Sys/Initialize.cs

```diff
@@ -47,7 +51,12 @@ public IValue PlainValue
                         s.Select(
                             kv => new KeyValuePair<IKey, IValue>(
                                 (Binary)kv.Key.Bencoded,
-                                kv.Value
+                                new Dictionary(kv.Value.Select(
+                                    kv2 => new KeyValuePair<IKey, IValue>(
+                                        (Binary)kv2.Key.Bencoded,
+                                        kv2.Value
+                                    )
+                                ))
```

Comment:
Thought about this a bit, and I'm of the opinion that this should stay intact. Unless we need it in the near future, I don't think this should change without a consensus that this is a necessary design change. We need a proper discussion on what constitutes a "system action". 😐